torch/_inductor/fx_passes/pad_mm.py

```diff
@@ -485,7 +481,47 @@ def write_pad():
                 )
             )
 
-        pad_time = do_bench(lambda: [fn() for fn in fns])
+        def orig_bench_fn():
+            if op is torch.ops.aten.bmm or op is torch.ops.aten.mm:
+                op(mat1, mat2)
+            else:
+                op(input, mat1, mat2)
+
+        def pad_bench_fn():
+            for fn in fns:
+                fn()
+
+        pad_time = None
+        if (
```

Comment:
What do we do if this path isn't set? Where does the padding time come from?

Reply:
It will be set here https://github.com/pytorch/pytorch/pull/128643/files/3e8d824835a4cda51232a8b61a545d4874583ac6#diff-368af613ae57b7883e87eb246be513c089e729d0e5d09bd1ea9a40d7b8140ec3R524